README.rst

```diff
@@ -12,6 +12,15 @@ Driver Mantras
 When developing specifications -- and the drivers themselves -- we follow the
 following principles:
 
+Strive to be idiomatic, but favor consistency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Drivers attempt to provide the easiest way to work with MongoDB in a given 
+language ecosystem while specifications attempt to provide a consistent 
+behavior and experience across all languages. Drivers should strive to be as 
+idiomatic as possible while meeting the specification and staying true to the 
+original intent.
```

Comment:
Can you comment where the original intent is likely to have been expressed? Is it the motivation section in the scope doc?

Reply:
Specifications already contain language that allow drivers to respect language idioms where appropriate. Naming is the most common area where this shows up, but I don't recall any specs that call out idiomaticity as a general motivation. This mantra seems more geared towards driver design than written specs. As a spec mantra, perhaps this should state that specs should be selective about requiring something for consistency (e.g. MUST) vs. allowing drivers to follow their own idioms -- and our prior art for naming variations of allowing stylistic differences (e.g. underscores, camelCase) while requiring specific terms can be referenced.

Re: scopes, I don't think those docs are relevant to this repository, since they're private and lose importance once we have a published specification.

Reply:
Good point. Maybe we can modify the final sentence:
```
Drivers should strive to be as idiomatic as possible while meeting the specification. 
Specifications should be selective about prescribing driver behavior and/or requiring
the adoption of specific names in driver APIs as doing so often compromises idiomaticity
for consistency.
```

Reply:
As pointed out by Bernie, my suggestion directly conflicts with the title of this section - `Strive to be idiomatic, but favor consistency`. I withdraw my comment/objection! 😸